xml/ssh_banners.xml

```diff
@@ -2125,6 +2125,18 @@
     <param pos="0" name="service.product" value="WeOnlyDo SSH Server"/>
   </fingerprint>
 
+  <fingerprint pattern="^Zyxel SSH server$">
+    <description>Zyxel Firewall SSH service</description>
+    <example>Zyxel SSH server</example>
+    <param pos="0" name="service.vendor" value="Zyxel"/>
+    <param pos="0" name="service.family" value="Zywall"/>
+    <param pos="0" name="os.vendor" value="Zyxel"/>
+    <param pos="0" name="os.product" value="ZyNOS"/>
```

Comment:
If you change this value to `ZyNOS Firmware` then `update_cpes.py` (https://github.com/rapid7/recog/blob/master/CONTRIBUTING.md#updating-cpes) will auto populate the CPE.

Reply:
completed this change and ran update_cpes